src/api/tigl.h

```diff
@@ -4493,6 +4493,40 @@ TIGL_COMMON_EXPORT TiglReturnCode tiglFuselageGetSegmentVolume(TiglCPACSConfigur
  */
 /*@{*/
 
+
+
+
+
+/**
+ * @brief tiglGetCrossSectionArea returns the area of the intersection of either the fused aircraft or its component (via UID) with a plane.
+ * The function can be used for Whitcomb's area rule.// If no intersections are found, the resulting area
+ *                                                   //is zero. No warning or error is thrwon in this case. If more than one intersection is found, the returned area
+ *                                                   //is the sum of the areas of the intersections. No warning or error is thrown.
```

Comment:
```suggestion
 * The function can be used for Whitcomb's area rule.
 *
 * If no intersections are found, the resulting area is zero. No warning or error is thrown in this case. 
 * If more than one intersection is found, the returned area is the sum of the areas of the intersections. No warning or error is thrown.
```

Note, apart from the type (thrwon -> thrown), I am not sure how doxygen would render the documentation with the `//` parts.

Reply:
Also: Can you please mention in the documentation, that if the uid of the whole airplane is given, that the airplane will be fused before calculating the intersections?

Reply:
Just added, thanks!